sbi/neural_nets/embedding_nets.py

```diff
@@ -236,24 +242,23 @@ def __init__(
                 Remark: This network should be large enough as it acts on all (iid)
                 inputs seperatley and needs enough capacity to process the information
                 of all inputs.
-            trial_net_output_dim: Dimensionality of the output of the trial_net / input
-                to the fully connected layers.
-            combining_operation: How to combine the permutational dimensions, one of
-                'mean' or 'sum'.
+            trial_net_output_dim: Dimensionality of the output of the trial_net.
+            aggregation_dim: Dimension along which to aggregate the trial embeddings.
+            aggregation_fn: Function to aggregate the trial embeddings, e.g.,
+                torch.mean, torch.sum, must be permutation invariant.
+                Defaults to taking the mean over the non-nan values.
```

Comment:
I think i would prefer the sum as default since it is inherently amortized over the number of trials

Reply:
fixed.